### PR TITLE
feat: [MSSQL] expose publicly_accessible property

### PR DESCRIPTION
**Pull request by fnaranjo-vmw**
[#185201362]

### Checklist:

* [x] Have you added Release Notes in the docs repositories?
* [x] Have you ran `make run-integration-tests` and `make run-terraform-tests`?
* [ ] Have you ran acceptance tests for the service under change?
* [x] Have you followed the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)?

### CODE REVIEW

File: integration-tests/mssql_test.go [hunk 1]
```diff
@@ -53,6 +53,7 @@ var _ = Describe("MSSQL", Label("MSSQL"), func() {
 			"rds_subnet_group":           "some-rds-subnet-group",
 			"instance_class":             "some-instance-class",
 			"max_allocated_storage":      999,
+			"publicly_accessible":        false,
```
Comment (zucchinidev) [inline integration-tests/mssql_test.go:56]: nit-pick: this property is not needed. The default value is propagate

**Review by zucchinidev — CHANGES_REQUESTED** [review]
Great job @fnaranjo-vmw . I asked for a little refactor

File: integration-tests/mssql_test.go (hunk 1 above)
Comment (fnaranjo-vmw) [reply integration-tests/mssql_test.go:56]: Feel free to remove it. I have the tendency to add every new field to some of the maps in the tests (required, optional, defaults, etc.)

Comment (zucchinidev) [inline integration-tests/mssql_test.go:56]:
```suggestion
````

**Review by zucchinidev — COMMENTED** [review]
I added the suggested change

**Review by zucchinidev — APPROVED** [review]

**Pull request merged**